apps/frontend/src/components/global/Sidebar.vue

```diff
@@ -1,65 +1,419 @@
-        :files="visible_profile_files"
-        :all-selected="all_profiles_selected"
-        @toggle-all="toggle_all_profiles"
-        @changed-files="$emit('changed-files')"
-      />
-    </v-expansion-panels>
-  </v-navigation-drawer>
+  <div>
+    <ChecklistTargetDataModal
+      :visible="showTargetModal"
+      style="display: none"
+      @close-modal="showTargetModal = false"
+    />
+    <ChecklistTechnologyAreaModal
+      :visible="showTechnologyModal"
+      style="display: none"
+      @close-modal="showTechnologyModal = false"
+    />
+
+    <v-navigation-drawer
+      :clipped="$vuetify.breakpoint.lgAndUp"
+      app
+      :style="{'z-index': 11}"
+      disable-resize-watcher
+      disable-route-watcher
+      fixed
+      permanent
+      width="45px"
+      @input="$emit('input', $event)"
+      @blur="value = false"
+    >
+      <v-container v-if="!isSideUtilityDrawerShown" fill-height fluid>
+        <v-row align="center" justify="center">
+          <v-col
+            ><v-icon
+              class="text-center"
+              right
+              @click="isSideUtilityDrawerShown = !isSideUtilityDrawerShown"
+              >mdi-arrow-right</v-icon
+            ></v-col
+          >
+        </v-row>
+      </v-container>
+    </v-navigation-drawer>
+    <v-navigation-drawer
+      v-model="isSideUtilityDrawerShown"
+      :clipped="$vuetify.breakpoint.lgAndUp"
+      app
+      :style="{'z-index': 11, 'margin-top': classification ? '5em' : '3em'}"
+      disable-resize-watcher
+      disable-route-watcher
+      fixed
+      temporary
+      width="600px"
+      @input="$emit('input', $event)"
+      @blur="value = false"
+    >
+      <div v-if="isSideUtilityDrawerShown">
+        <v-expansion-panels v-model="active_path" accordion>
+          <DropdownContent
+            header-text="Results"
+            :files="visible_evaluation_files"
+            :all-selected="all_evaluations_selected"
+            :enable-compare-view="true"
+            :compare-view-active="compareViewActive"
+            @toggle-all="toggle_all_evaluations"
+            @toggle-compare-view="compareView"
+            @changed-files="$emit('changed-files')"
+          />
+          <DropdownContent
+            header-text="Profiles"
+            :files="visible_profile_files"
+            :all-selected="all_profiles_selected"
+            @toggle-all="toggle_all_profiles"
+            @changed-files="$emit('changed-files')"
+          />
+          <DropdownContent
+            header-text="Checklists"
+            :files="visible_checklist_files"
+            :all-selected="all_checklists_selected"
+            @toggle-all="toggle_all_checklists"
+            @changed-files="$emit('changed-files')"
+          />
+        </v-expansion-panels>
+        <div class="mx-5 mr-10 mb-5">
+          <v-divider class="mb-5" />
+          <!-- Checklist Data Modals -->
+          <v-row v-if="inChecklistView()" class="my-4" style="width: 600px">
```

Comment:
i think this should be a watcher or similar variable, not an explicit function call

Reply:
Converted to a watcher